CHANGELOG.md

```diff
@@ -11,6 +11,9 @@ The Dovecot and Postfix overrides are moved in their own sub-directory.
 If there are local override files, they will need to be moved from overrides/ to overrides/dovecot and overrides/postfix/.
 See https://mailu.io/1.8/faq.html#how-can-i-override-settings for all the mappings.
 
+Resolution of HOST_* environment variables at startup, and the corresponding *_ADDRESS variables have been removed.
+Use the HOST_* variables directly instead.
+
```

Comment:
This change would be in an eventual 1.9, not 1.8. Not sure if it makes sense to clear the entire section when releasing to add new entries like this or alternatively to change it in a single commit when doing the release. Either way currently it's mixed.